docs/pages/server-access/guides/openssh.mdx

```diff
@@ -340,6 +340,71 @@ authenticate the host via the certificate we generated earlier.
 
 </Details>
 
+<Details title="Using Proxy Jump in Trusted Clusters?">
+  
+Normally, you connect to leaf cluster SSH nodes through the root cluster proxy,
```

Comment:
I think we can start this paragraph by referring to the SSH config generated above, rather than what normally happens—I'm not sure what "normally" means in this context. Maybe something like:

```markdown
The configuration file that `tsh config` generates uses a `ProxyCommand`
directive to connect to SSH servers in a leaf cluster through the root cluster's
Proxy Service.
```